app/src/main/res/values/strings.xml

```diff
@@ -35,12 +35,15 @@
     <string name="map_attribution_osm">"© OpenStreetMap contributors"</string>
     <string name="quest_generic_otherAnswers">"Other answers…"</string>
     <string name="quest_generic_answer_notApplicable">"Can’t say…"</string>
+    <string name="quest_generic_answer_notExisting">"Does not exist…"</string>
     <string name="quest_generic_confirmation_title">"Are you sure?"</string>
     <string name="quest_generic_confirmation_yes">"Yes, I am sure"</string>
     <string name="quest_generic_confirmation_no">"I will check"</string>
     <string name="quest_generic_error_field_empty">"You left this field empty"</string>
     <string name="quest_leave_new_note_title">"Leave a note instead?"</string>
     <string name="quest_leave_new_note_description">"You can leave a public note at this location which other mappers can comment on and hopefully resolve or alternatively just hide the quest for yourself only"</string>
+    <string name="quest_leave_new_note_request">"Please, leave a note"</string>
+    <string name="quest_leave_new_note_about_nonexisting_object_description">"Questions may be asked about objects that are not existing. It happens when OpenStreetMap contains wrong or outdated info. You can leave a public note at this location with explanation what is wrong or alternatively just hide the quest for yourself only."</string>
```

Comment:
The English could probably be improved by saying something like:
```suggestion
    <string name="quest_leave_new_note_about_nonexisting_object_description">"Questions may be asked about objects that no longer exist. It happens when OpenStreetMap contains incorrect or outdated info. You can leave a public note at this location with an explanation of what is wrong or alternatively just hide the quest for yourself only."</string>
```

Let me know if it needs to be a bit briefer and we could probably reword it slightly to shorten it.

notExisting for the variable isn't great, but given you're trying to say "doesn't exist" or "no longer existing" both could be a bit hard to camel case. Actually non-existent might be better (perhaps as nonExistent)?